ios/PurchasesHybridCommon/PurchasesHybridCommon/PaywallProxy.swift

```diff
@@ -53,6 +61,26 @@ import UIKit
         }
     }
 
+    @objc
+    public func presentPaywallIfNeeded(requiredEntitlementIdentifier: String, displayCloseButton: Bool) {
```

Comment:
Same for this one and the previous `presentPaywallIfNeeded` (into a different method)